### PR TITLE
fix for non-existing plugin remove

### PR DESCRIPTION
**Pull request by palinek**
Fixes lxde/lxqt#451.

### CODE REVIEW

**Comment by palinek** [comment]
How about this one guys?

**Comment by paulolieuthier** [comment]
I bookmarked it the first time I saw it. But didn't have time yet. Probably next week.

File: panel/lxqtpanel.cpp [hunk 1]
```diff
@@ -640,7 +635,8 @@ void LxQtPanel::showAddPluginDialog()
 void LxQtPanel::addPlugin(const LxQt::PluginInfo &desktopFile)
 {
     QString settingsGroup = findNewPluginSettingsGroup(desktopFile.id());
-    loadPlugin(desktopFile, settingsGroup);
+    if (0 != loadPlugin(desktopFile, settingsGroup))
```
Comment (paulolieuthier) [inline panel/lxqtpanel.cpp]: I don't understand this clause.

Comment (palinek) [inline panel/lxqtpanel.cpp]: It's there to leave previous functionality untouched (if loading of newly added plugin failed, nothing was added into *mPlugins* => no new plugin was added into settings). IMO this is correct behaviour.

Comment (paulolieuthier) [inline panel/lxqtpanel.cpp]: Oh, sorry, I thought it was `|=`, not `!=`. That's what very few hours of sleep get you.

**Comment by paulolieuthier** [comment]
Landed in lxde/lxqt@266a3c014e2469abac7304df245fc0f852eb1b18. Thanks, again, @palinek!

**Pull request closed without merging**